tests/test_distributed_circuit_breaker.py

```
@@ -0,0 +1,499 @@
+            failure_threshold=3,
+            timeout_seconds=0  # Immediate timeout
+        )
+        breaker = DistributedCircuitBreaker(mock_redis, "test_service", config=config)
+
+        # Open the circuit
+        for _ in range(4):
+            await breaker.record_failure()
+
+        # Now check can_execute - should transition to half-open after timeout
+        # Since timeout is 0, it should transition immediately
+        can_exec = await breaker.can_execute()
```

Ruff (0.14.14)

[error] Local variable `can_exec` is assigned to but never used

Remove assignment to unused variable `can_exec`

(F841)